java/arcs/android/host/parcelables/ParcelablePlan.kt

```diff
@@ -18,30 +18,48 @@ import arcs.core.data.Plan
 /** [Parcelable] variant of [Plan]. */
 data class ParcelablePlan(override val actual: Plan) : ActualParcelable<Plan> {
     override fun writeToParcel(parcel: Parcel, flags: Int) {
+        parcel.writeInt(actual.handles.size)
+        actual.handles.forEach {
```

Comment:
You don't _need_ to write them here, as you write copies for each handle connection,  right?

Reply:
True, they could be reconstructed from the HandleConnection copies I think. Don't write them, and on read, synthesize Plan.handles from mapping over HandleConnection.handle. That would solve the dup-bug

Reply:
removed.
there is still some duplication, as multiple particles may have the same handle, so keeping the bug.